<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupTotpDialog</class>
 <widget class="QDialog" name="SetupTotpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup TOTP</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Key:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="seedEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioDefault">
       <property name="text">
        <string>Default RFC 6238 token settings</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">settingsButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioSteam">
       <property name="text">
        <string>Steam token settings</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">settingsButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioCustom">
       <property name="text">
        <string>Use custom settings</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">settingsButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Note: Change these settings only if you know what you are doing.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <property name="rowWrapPolicy">
      <enum>QFormLayout::DontWrapRows</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="stepLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Time step:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="radio8Digits">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>8 digits</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="radio6Digits">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>6 digits</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="digitsLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Code size:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="stepSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string comment="Seconds"> sec</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="settingsButtonGroup"/>
 </buttongroups>
</ui>
